tracing/tests/no_subscriber.rs

```diff
@@ -1,14 +1,15 @@
 #![cfg(feature = "std")]
 
-use tracing::subscriber::{self, NoSubscriber};
+use tracing_mock::subscriber;
 
 #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test::wasm_bindgen_test)]
 #[test]
-fn no_subscriber_disables_global() {
+fn no_collector_disables_global() {
```

Comment:
let's use the v0.1.x naming for this please (`no_subscriber...` rather than `no_collector...`)

Reply:
whoops, missed this, sorry.